tests/spyre_util.py

```diff
@@ -528,11 +528,16 @@ def _get_or_default(env: str, default: str) -> str:
             "VLLM_SPYRE_TEST_MODEL_LIST",
             "sentence-transformers/all-roberta-large-v1")
         marks = [pytest.mark.embedding]
-    elif quantization == "gptq":
+    elif quantized == "gptq":
         # TODO: need a HF hub reference here as a default
         user_test_model_list = _get_or_default("VLLM_SPYRE_TEST_MODEL_LIST",
                                                "granite-3.0-8b-instruct-gptq")
         marks = [pytest.mark.decoder, pytest.mark.quantized, pytest.mark.spyre]
+    elif quantized == "fp8":
+        user_test_model_list = _get_or_default(
+            "VLLM_SPYRE_TEST_MODEL_LIST",
+            "ibm-ai-platform/micro-g3.3-8b-instruct-1b-FP8")
+        marks = [pytest.mark.decoder, pytest.mark.quantized]
```

Comment:
do we need to add `spyre` here?

Reply:
I don't think so? Maybe the gptq one had it because it didn't work on cpu?

Reply:
Verified

```
❯ VLLM_SPYRE_TEST_BACKEND_LIST=sendnn pytest --collect-only -m "spyre and quantized" tests
<Dir vllm-spyre>
  <Dir tests>
    <Dir e2e>
      <Module test_spyre_online.py>
        <Function test_openai_serving_gptq[warmup_shape0-gptq-sendnn-ibm-ai-platform/micro-g3.3-8b-instruct-1b]>
        <Function test_openai_serving_fp8[warmup_shape0-sendnn-ibm-ai-platform/micro-g3.3-8b-instruct-1b]>
```